src/org/spdx/licenselistpublisher/licensegenerator/LicenseRdfFormatWriter.java

```diff
@@ -103,7 +103,7 @@ public void setRdfNt(File rdfNt) {
 
 	@Override
 	public void writeLicense(SpdxListedLicense license, boolean deprecated, String deprecatedVersion) throws IOException, LicenseGeneratorException {
-		license.setCrossRef(UrlHelper.buildUrlDetails(license.getSeeAlso()));
+		license.setCrossRef(CrossRefHelper.buildUrlDetails(license.getSeeAlso()));
```

Comment:
These would be removed since it is built in the builder not in the writers per my previous comment